### PR TITLE
[DOCS] Revise GS intro & remove redundant conceptual info. Closes #43846

### LINKED ISSUE #43846: [DOCS] Streamline the Elasticsearch Getting Started intro
Rework the intro and remove the information that's redundant with the Elasticsearch intro. Part of #43721.

### PR DESCRIPTION
**Pull request by debadair**
Step 1 in streamlining the Elasticsearch getting started tutorial. Subsequent changes will better align the GS topics with the intro.

### CODE REVIEW

File: docs/reference/getting-started.asciidoc [hunk 1]
```diff
@@ -1,107 +1,33 @@
 [[getting-started]]
-= Getting started
+= Getting started with {es}
```
Comment (jrodewig) [inline docs/reference/getting-started.asciidoc]: A title abbrev might be helpful here.

```
++++
<titleabbrev>Getting started</titleabbrev>
++++
```

File: docs/reference/getting-started.asciidoc [hunk 2]
```diff
@@ -1,107 +1,33 @@
...
 
 [partintro]
 --
-TIP: The fastest way to get started with {es} is to https://www.elastic.co/cloud/elasticsearch-service/signup[start a free 14-day trial of Elasticsearch Service] in the cloud.
+Ready to take {es} for a test drive and see for yourself how you can use the
+REST APIs to store, search, and analyze data?
 
-Elasticsearch is a highly scalable open-source full-text search and analytics engine. It allows you to store, search, and analyze big volumes of data quickly and in near real time. It is generally used as the underlying engine/technology that powers applications that have complex search features and requirements.
+Step through this getting started tutorial to:
```
Comment (jrodewig) [inline docs/reference/getting-started.asciidoc]: "Step through" struck me as odd wording.

File: docs/reference/docs/data-replication.asciidoc [hunk 3]
```diff
@@ -5,12 +5,12 @@
 [float]
 === Introduction
 
-Each index in Elasticsearch is <<getting-started-shards-and-replicas,divided into shards>>
+Each index in Elasticsearch is <<scalability,divided into shards>>
```
Comment (jrodewig) [inline docs/reference/docs/data-replication.asciidoc]:
```suggestion
Each index in {es} is <<scalability,divided into shards>>
```

File: docs/reference/docs/data-replication.asciidoc [hunk 4]
```diff
@@ -5,12 +5,12 @@
...
 and each shard can have multiple copies. These copies are known as a _replication group_ and must be kept in sync when documents
 are added or removed. If we fail to do so, reading from one copy will result in very different results than reading from another.
 The process of keeping the shard copies in sync and serving reads from them is what we call the _data replication model_.
 
-Elasticsearch’s data replication model is based on the _primary-backup model_ and is described very well in the 
+Elasticsearch’s data replication model is based on the _primary-backup model_ and is described very well in the
```
Comment (jrodewig) [inline docs/reference/docs/data-replication.asciidoc]:
```suggestion
{es}'s data replication model is based on the _primary-backup model_ and is described very well in the
```

We may want to consider replacing emphasis (italics) here with bold. I think it's a bit more readable.

File: docs/reference/docs/data-replication.asciidoc [hunk 5]
```diff
@@ -23,7 +23,7 @@ it has for various interactions between write and read operations.
 [float]
 === Basic write model
 
-Every indexing operation in Elasticsearch is first resolved to a replication group using <<index-routing,routing>>, 
+Every indexing operation in Elasticsearch is first resolved to a replication group using <<index-routing,routing>>,
```
Comment (jrodewig) [inline docs/reference/docs/data-replication.asciidoc]:
```suggestion
Every indexing operation in {es} is first resolved to a replication group using <<index-routing,routing>>,
```
Passive.

File: docs/reference/docs/data-replication.asciidoc [hunk 6]
```diff
@@ -153,8 +153,8 @@ Dirty reads:: An isolated primary can expose writes that will not be acknowledge
 [float]
 === The Tip of the Iceberg
 
-This document provides a high level overview of how Elasticsearch deals with data. Of course, there is much much more 
-going on under the hood. Things like primary terms, cluster state publishing, and master election all play a role in 
+This document provides a high level overview of how Elasticsearch deals with data. Of course, there is much much more
```
Comment (jrodewig) [inline docs/reference/docs/data-replication.asciidoc]:
```suggestion
This document provides a high level overview of how {es} deals with data. Of course, there is more
```

**Review by jrodewig — APPROVED** [review]
Some minor nits but nothing blocking. LGTM.

File: docs/reference/docs/data-replication.asciidoc (hunk 3 above)
Comment (debadair) [reply docs/reference/docs/data-replication.asciidoc]: I was trying to leave the blinders on when I fixed the xref...

Comment (jrodewig) [reply docs/reference/docs/data-replication.asciidoc]: Fair enough. Feel free to ignore. Not trying to inflate scope.

Comment (jrodewig) [reply docs/reference/docs/data-replication.asciidoc]: Ah. Took another look. It looks like your text editor might be trimming whitespace, which makes the changes look larger than they are. My mistake.

I'll resolve any comments I have in data-replication.asciidoc.

**Pull request merged**